<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interlock</class>
 <widget class="QWidget" name="Interlock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="SetMaxSpanInter">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="MaxSpanInput">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>62</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-10000000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="MinSpanInput">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>62</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000000000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="MaxForceInput">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>62</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000000000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Span</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min Span</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Force</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="MinForceInput">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>62</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-100000000000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min Force</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetMinSpanInter">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetMaxForceInter">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetMinForceInter">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>E-Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UnlockEStop">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UnlockMaxSpan">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Unlock</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UnlockMinSpan">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Unlock</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UnlockMaxForce">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Unlock</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UnlockMinForce">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Unlock</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Controller</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ControllerSelect">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>43</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ON?</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="MaxDispCheckBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="MinDispCheckBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="MaxForceCheckBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="MinForceCheckBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Act 1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="MaxForceCheckBox_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="MinForceCheckBox_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Act 2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="MaxDispCheckBox_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="MinDispCheckBox_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="EStopCheckBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
